src/model/guild/guild_id.rs

```diff
@@ -1069,6 +1047,169 @@ impl GuildId {
     ) -> ReactionCollectorBuilder<'a> {
         ReactionCollectorBuilder::new(shard_messenger).guild_id(self.0)
     }
+
+    /// Creates a guild specific [`ApplicationCommand`]
+    ///
+    /// **Note**: Unlike global `ApplicationCommand`s, guild commands will update instantly.
+    ///
+    /// # Errors
+    ///
+    /// Returns the same possible errors as `create_global_application_command`.
+    ///
+    /// [`ApplicationCommand`]: crate::model::interactions::ApplicationCommand
```

Comment:
This should link to the function.
```suggestion
    /// Returns the same possible errors as [`create_global_application_command`].
    ///
    /// [`ApplicationCommand`]: crate::model::interactions::ApplicationCommand
    /// [`create_global_application_command`]: Self::create_global_application_command
```